src/FsCheck/paket.template

```diff
@@ -25,6 +25,8 @@ description
     Properties are written in F#, C# or VB. FsCheck helps you to define properties, observe the distribution of test data, and define test data generators. Crucially, when a property fails, FsCheck automatically displays a minimal counter example.
 dependencies
     FSharp.Core >= 4.1.0
+	framework: net452
+		FSharp.Core >= 4.1.0
```

Comment:
Try FSharp.Core 4.1.18 here? Earlier versions of FSharp.Core pick up extra packages https://github.com/Microsoft/visualfsharp/issues/3451

Reply:
Can't do that: 4.1.17 is the last FSharp.Core to have PCL builds.

Reply:
Fair enough.